resources_pages/block_rep.md

```diff
@@ -1,40 +1,40 @@
 ---
 layout: page
-title: Block Representatives
+title: Student Representatives
 ---
 
-## MDS Block Representatives
+## MDS Student Representatives
 
-In the new academic year, there will be one set of block representatives in Term 1 (Blocks 1-3) and another set in Term 2 (Blocks 4-6). Block representatives volunteer to represent their peers, acting as a liaison between students and MDS leadership. This allows us to monitor how each block is progressing and make adjustments if needed.
+In the new academic year, there will be one set of student representatives in Term 1 (Blocks 1-3) and another set in Term 2 (Blocks 4-6). Student representatives volunteer to represent their peers, acting as a liaison between students and MDS leadership. This allows us to monitor how the program is progressing and make adjustments if needed.
```

Comment:
There will also be a third set of student reps during capstone, so that should be included here as well.

Reply:
@katieburak , shall we also enumerate the five pillars (_academic_, _social_, _sports and recreation_, _career and professional development_, and _mental health and wellness_)?

Reply:
Agree!